src/schemas/out-of-bc-address-schema.ts

```diff
@@ -0,0 +1,33 @@
+import { required, maxLength } from 'vuelidate/lib/validators'
+import { AddressSchemaIF } from '@/interfaces/'
+
+// The Office Address schema containing Vuelidate rules.
+// NB: This should match the subject JSON schema.
+export const OutOfBCAddressSchema: AddressSchemaIF = {
+  streetAddress: {
+    required,
+    maxLength: maxLength(50)
+  },
+  streetAddressAdditional: {
+    maxLength: maxLength(50)
+  },
+  addressCity: {
+    required,
+    maxLength: maxLength(40)
+  },
+  addressCountry: {
+    required,
+    // FUTURE: create new validation function isCountry('CA')
+    isCanada: (val: string) => Boolean(val === 'CA')
+  },
+  addressRegion: {
+    maxLength: maxLength(2)
```

Comment:
Shouldnt this have another rule here that checks region not equals BC?

Reply:
Yes Lekshmi. There is another rule, but that rule is for business inside BC. For this ticket, when adding a business partner that is Unregistered in B.C., we don't want the form to throw the error, so I removed the validation check for BC. Since this is a schema, the best way I could think of was to make another schema to avoid the BC validation. That's why I made this out-of-bc-address-schema ts file.

Reply:
ok